pkg/trace/stats/quantile/converter_test.go

```diff
@@ -127,3 +143,7 @@ func TestDDToGKSketch(t *testing.T) {
 	t.Run("constant10", testDDSketchToGKConstant(10))
 	t.Run("constant1e3", testDDSketchToGKConstant(1000))
 }
+
+func TestDecoding(t *testing.T) {
+	t.Run("dense_sketch_from_Java", testUnmarshalDenseSketchFromFile("java_generated_dense_ddsketch.ddsketch", 101, 631))
```

Comment:
```suggestion
	t.Run("dense", testUnmarshalDenseSketchFromFile("java_generated_dense_ddsketch.ddsketch", 101, 631))
```
Nit. We usually use one-word, when possible in Go. Go is a bit different from languages where test names describe the test. Here it's more of an identifier. You can go longer with the function name if you think that helps, maybe `TestDecodeDDSketch` or something? Just a suggestion.